code/game/objects/items/storage/fancy.dm

```diff
@@ -81,9 +82,34 @@
 	for(var/i in 1 to 6)
 		new /obj/item/reagent_containers/food/snacks/donut(src)
```

Comment:
```suggestion
		new spawn_type(src)
```
If you do this, you can remove ``PopulateContents`` from the deadly donut box too, making it also use parent (therefore harder for it to get bugs)